File: CS/WorkflowServerService2/ServerApplication.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.ExpressApp;

namespace WorkflowServerService2 {
    public class ServerApplication : XafApplication {
        protected override DevExpress.ExpressApp.Layout.LayoutManager CreateLayoutManagerCore(bool simple) {
            throw new NotImplementedException();
        }
        public void Logon() {
            base.Logon(null);
        }
    }
}

File: VB/WorkflowServerService2/ServerApplication.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.ExpressApp

Namespace WorkflowServerService2
	Public Class ServerApplication
		Inherits XafApplication

		Protected Overrides Function CreateLayoutManagerCore(ByVal simple As Boolean) As DevExpress.ExpressApp.Layout.LayoutManager
			Throw New NotImplementedException()
		End Function
		Public Sub Logon()
			MyBase.Logon(Nothing)
		End Sub
	End Class
End Namespace
